src/sidekiq/pool.cr

```diff
@@ -62,7 +64,7 @@ module Sidekiq
     end
 
     def initialize(redis_cfg : RedisConfig)
-      @pool = ConnectionPool(Redis).new(capacity: redis_cfg.pool_size, timeout: redis_cfg.pool_timeout) do
+      @pool = DB::Pool.new(checkout_timeout: redis_cfg.pool_timeout, max_pool_size: redis_cfg.pool_size) do
```

Comment:
Why this needed? Redis client already have internal pool of connections (PooledClient), i think this can be removed.

Reply:
I didn't know, but perfect! I would love to, could you provide a PR?

Reply:
I have a patch for this: https://github.com/hugopl/sidekiq.cr/commit/a73e50b99d784ed2137988e42c203ead7fd5eb2f

I'm just waiting to see if kemal/kemal-csrf will be released soon, since it doesn't wotk with kemal 1.0.